modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -170,8 +170,10 @@ public interface MetaStorageManager extends IgniteComponent {
      * Starts all registered watches.
      *
      * <p>Should be called after all Ignite components have registered required watches and they are ready to process Meta Storage events.
+     *
+     * @return Future to complete.
```

Comment:
Please add meaningful comment here.

Reply:
You're joking right? The core idea of the current ticket (IGNITE-19606) is to linearise ms.start() and ms.deployWatches().
Do we still have an implementation with unconnected start and deployWatches?